docs/documentation/e2e.md

```diff
@@ -14,16 +14,72 @@ ng e2e
 End-to-end tests are run via [Protractor](https://angular.github.io/protractor/).
 
 ## Options
+`--aot` Build using Ahead of Time compilation.
```

Comment:
e2e inherits options from serve, can you list just the e2e specific options (the ones in the command file) and refer to the serve command for the rest?